### DISCUSSION TITLE
Advice on minio configuration (new machine)

### DISCUSSION THREAD

**Discussion by JoyMonteiro** [minio/minio#12524] [Q&A] [answered]
Hello All,

we have a server with 36 HDDs each of 14 TB (~500 TB). we need to store a few different kinds of data, and wanted advice on how best to configure minio. This is the first time I'm configuring anything of this kind (HA object storage).

These are 3 kinds of data:

1. Static earth observation data. It is going to take up most of the space on the server. However, it is straightforward (if somewhat tedious) to download in case something goes wrong.
2. Simulation data from models. This will take up ~10-15% of the space, but it is not easy to get back.
3. Temporary files generated during exploratory data analysis. This will take up the least space, but will be often changed/deleted.

My initial feeling is that I would like to store 1) with lesser redundancy, 2) with standard redundancy and use zfs for 3).

Any suggestions whether this makes sense? If it does, any suggestions on how to configure minio to handle 1) and 2)?

We would also like to use both ethernet ports on the server to maximize throughput. Should I just use nginx and multiple
minio instances on the same disk array?

TIA,
Joy

**Comment by klauspost** [comment, marked as answer]
@JoyMonteiro You can write `hello@minio.io` or look directly at our [support options](https://min.io/pricing) so we can help you directly with making a good setup.

**Reply by JoyMonteiro** [reply]
Thanks, Klaus. As an academic from India, I don't think I can afford commercial support! Hence, I'm asking the community here.

I'm quite familiar with system administration, just that I haven't setup minio before.
